### PR TITLE
terratest

### PR DESCRIPTION
**Pull request by pjdufour-dds**

### CODE REVIEW

File: test/terraform_aws_encrypted_test.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// =================================================================
+//
+// Work of the U.S. Department of Defense, Defense Digital Service.
+// Released as open source under the MIT License.  See LICENSE file.
+//
+// =================================================================
```
Comment (chrisgilmerproj) [inline test/terraform_aws_encrypted_test.go:6]: We really should make a pre-commit hook to add this to files. Then we can have it auto-add for us to new files.

**Review by chrisgilmerproj — APPROVED** [review]
🚀 - this is really cool stuff!

File: test/terraform_aws_encrypted_test.go (hunk 1 above)
Comment (pjdufour-dds) [reply test/terraform_aws_encrypted_test.go:6]: I like that idea, but would like to get this merged first.

Comment (chrisgilmerproj) [reply test/terraform_aws_encrypted_test.go:6]: yep, not a blocker. Just something to keep in mind for the future.

**Pull request merged**